### PR TITLE
Never shorten signers

### PR DESCRIPTION
**Pull request by soltysh**
/assign @deads2k

### CODE REVIEW

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 1]
```diff
@@ -72,20 +81,33 @@ func NewCertRotationController(
 	configInformer.Config().V1().Networks().Informer().AddEventHandler(ret.serviceHostnameEventHandler())
 	configInformer.Config().V1().Infrastructures().Informer().AddEventHandler(ret.externalLoadBalancerHostnameEventHandler())
 
-	rotationDay := defaultRotationDay
```
Comment (deads2k) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]:
```
neverShortenedDay := defaultRotationDay
...




```

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 2]
```diff
@@ -72,20 +81,33 @@ func NewCertRotationController(
...
+		}
+		klog.Warningf("!!! UNSUPPORTED VALUE SET !!!")
 	}
 
 	certRotator, err := certrotation.NewCertRotationController(
 		"AggregatorProxyClientCert",
 		certrotation.SigningRotation{
 			Namespace:     operatorclient.OperatorNamespace,
 			Name:          "aggregator-client-signer",
-			Validity:      30 * rotationDay,
```
Comment (deads2k) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]: `neverShortenedDay`

**Comment by soltysh** [comment]
Now with all signers, tagging back.

**Pull request merged**